mypy/mro.py

```diff
@@ -43,7 +43,7 @@ def linearize_hierarchy(info: TypeInfo,
 
 
 def merge(seqs: List[List[TypeInfo]]) -> List[TypeInfo]:
-    seqs = [s[:] for s in seqs]
+    seqs = [s.copy() for s in seqs]
```

Comment:
Seems unnecessary

Reply:
Yeah, doesn't really change anything. I thought it would make more readable though.